### ISSUE TITLE
Ga.ReciprocalFrame is nonsensical

### ISSUE THREAD

**Issue by eric-wieser** [pygae/galgebra#213]
This behavior doesn't make much sense to me:
```
>>> g, *_ = Ga.build('e*0|1|2')
>>> g.ReciprocalFrame(g.basis)
(1, -1, (e_1.e_2)/(e_2.e_2) + e_1^e_2/(e_2.e_2))
>>> g.ReciprocalFrame(g.basis, mode='no-norm-please')  # any string will do
((e_2.e_2)*1**2,
 -(e_2.e_2)*1**2,
 (e_1.e_2)*1**2 + 1**2*e_1^e_2,
 (e_2.e_2)*1**2)
```
I was expecting to get some vector elements in this output, not a mixture of scalars and bivectors.

**Comment by utensil** [comment]
The second output certainly should be corrected. What's you exact expected output in the first one?

**Comment by eric-wieser** [comment]
This issue was caused by using `basis` and not `mv_basis`

**Comment by utensil** [comment]
Is this completely addressed by #219 ? @eric-wieser

**Comment by eric-wieser** [comment]
Nope, added a checkbox for the bullet points above. `g.ReciprocalFrame(g.basis)` ought to either error, which is caused by `g.mv_basis[0] ^ g.basis[1]` behaving poorly

**Comment by eric-wieser** [comment]
Removed the `docs` tag, since this now has [a docstring and type annotations](https://galgebra.readthedocs.io/en/latest/generated/galgebra.ga.html#galgebra.ga.Sm.ReciprocalFrame).
